### PR TITLE
Database Migrations

### PR DESCRIPTION
**Pull request by janrebpol**
**Overview**
Update the tables in the database

**SNIPPETS**

### CODE REVIEW

File: src/routes/web.php [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
+// use Illuminate\Support\Facades\Route;
+// use App\Http\Controllers\SigninController;
+// use App\Http\Controllers\SignupController;
+// use App\Models\Profile;
+
+
+// Route::get('/', [
+//     SignupController::class,
+//      'showSignup']);
+
```
Comment (mikco-hipe) [inline src/routes/web.php]: Why did you comment out the routes?

File: src/database/migrations/2024_04_24_060621_create_user_profiles_table.php [hunk 2]
```diff
@@ -6,24 +6,23 @@
...
-    /**
+   /**
      * Run the migrations.
      */
     public function up(): void
     {
         Schema::create('profiles', function (Blueprint $table) {
             $table->id();
-            $table->unsignedBigInteger('user_id');
+            $table->unsignedBigInteger('user_id')->index();
```
Comment (mikco-hipe) [inline src/database/migrations/2024_04_24_060621_create_user_profiles_table.php:16]: It's not advisable to edit the migration file that was already been migrated. If you want to add, remove, rename a column, do so by adding another migration file.

ref:
https://laracoding.com/how-to-change-a-table-column-type-using-laravel-migrations/

File: src/app/Models/User.php [hunk 3]
```diff
@@ -46,3 +45,8 @@ protected function casts(): array
         ];
     }
 }
+
+public function profile(): HasOne
```
Comment (hipe01mark) [inline src/app/Models/User.php:52]: Add docbloc

**Review by mikco-hipe — APPROVED** [review]
LGTM 🙇

**Review by sunkenquest — APPROVED** [review]

**Pull request merged**